internal/js/modules/k6/browser/tests/tracing_test.go

```diff
@@ -263,7 +263,13 @@ func TestNavigationSpanCreation(t *testing.T) {
 				await Promise.all([
 					page.waitForNavigation(),
 					page.evaluate(() => window.history.back()),
-				]);
+				]).catch(e => {
+					// only throw exception if it's not related to navigational
+					// race condition, which can happen in slow machines.
+					if (!e.toString().includes('Inspected target navigated or closed')) {
+						throw e;
+					}
+				});
```

Comment:
Without this, this PR's CI jobs fail all the time.

This is a navigational issue that occurs when the code (i.e., `window.history.back`) executes on Chrome, causing the page to navigate back before the CDP's response is delivered to us, resulting in unnecessary noise. This error also occurred on CI previously, but it has become more apparent now due to our involved browser tests. I can open up a new PR just for this change, but I think it's small enough just to include it here.

Reply:
Added a separate PR here: #5378